db/migrate/20150617114935_create_email_requested_receipts.rb

```diff
@@ -0,0 +1,32 @@
+class CreateEmailRequestedReceipts < ActiveRecord::Migration
+  def change
+    create_table :email_requested_receipts do |t|
+      t.references :petition, index: true, foreign_key: true
```

Comment:
Just out of interest, I've been thinking to try PostgreSQL's `jsonb` datatype with Rails (seems that the support is pretty good with v. 4.2) some day - what do you think, could it suit this use case? Seems that the receipt implementation is very dynamic apart from the db side.